src/modules/governance/services/governance.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
+import { GovernanceContract } from '../models/governance.contract.model';
+import { governanceContractsAddresses } from '../../../utils/governance';
+import { GovernanceContractsFiltersArgs } from '../models/contracts.filter.args';
+
+@Injectable()
+export class GovernanceService {
+    async getGovernanceContracts(filters: GovernanceContractsFiltersArgs): Promise<GovernanceContract[]> {
```

Comment:
`filters` argument isn't used

Reply:
it will be used in the following implementation